app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id'
+  has_many :comments
+  has_many :likes
```

Comment:
- [ ] Dear @dieum512, for consistency and to stay aligned with the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png), Please apply the `foreign_key: 'author_id'` for the `Comment` and `Like` associations too.